packages/augur-ui/src/modules/events/actions/log-handlers.ts

```diff
@@ -486,10 +487,18 @@ export const handleMarketFinalizedLog = (log: Logs.MarketFinalizedLog) => (
   const { universe } = getState();
   if (universe.forkingInfo) {
     if (log.market === universe.forkingInfo.forkingMarket) {
-      dispatch(loadUniverseForkingInfo())
+      dispatch(loadUniverseForkingInfo());
     }
   }
-}
+  const { accountPositions } = getState();
```

Comment:
Can you put that alongside universe on line 487? Or would that affect the results in case of a fork?

Reply:
yup can do